doc/overview.md

```diff
@@ -0,0 +1,142 @@
+# APIcast Overview
+
+APIcast is an NGINX based API gateway used to integrate your internal and external API services with 3scale’s API Management Platform.
+
+Here you’ll learn more about deployment options, environments provided, and how to get started.
+
+## Deployment options
+
+You can use APIcast hosted or self-managed, in both cases, it needs connection to the rest of the 3scale API management platform:
+- **<a href="https://support.3scale.net/docs/deployment-options/apicast-cloud-gateway">APIcast hosted</a>**: 3scale hosts the gateway in the cloud. The API gateway is already deployed for you and it's limited to 50,000 calls per day.
+- **APIcast self-managed**: You can deploy APIcast wherever you want. To do so, download the json configuration file from API > Integration > Production or fetch it using the Account management API. The self-managed mode is the intended mode of operation for production environments. Here are a few recommended options to deploy your API gateway:
+  - <a href="https://support.3scale.net/docs/deployment-options/nginx-self-managed-setup">Local deploy</a>: Where the customer needs to deploy a supported version of Nginx/ OpenResty and use the 'GA' tagged release from the [APIcast repository](https://github.com/3scale/apicast/releases).

```

Comment:
the idea is that these documents will also eventually go into the 3scale support site hence the reference to GA, not sure what can be done in meantime for the document on this repo